config/src/test/java/com/alibaba/nacos/config/server/controller/ConfigOpsControllerTest.java

```diff
@@ -57,52 +57,51 @@
 @ContextConfiguration(classes = MockServletContext.class)
 @WebAppConfiguration
 public class ConfigOpsControllerTest {
-    
+
```

Comment:
Please use nacos code style to reformat codes

Reply:
It's done.